src/cdk/coercion/BUILD.bazel

```diff
@@ -1,14 +1,12 @@
 package(default_visibility=["//visibility:public"])
-load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
-load("@build_bazel_rules_typescript//:defs.bzl", "ts_library", "ts_web_test")
-load("@angular//:index.bzl", "ng_module")
 
+load("@build_bazel_rules_typescript//:defs.bzl", "ts_library", "ts_web_test")
 
-ng_module(
+ts_library(
```

Comment:
I believe that the `ng_package` rule only worked with this being an `ng_module`, but maybe that is moot now since we can't package the library without ivy anyway

Reply:
Just checked locally, and seems to work as expected.